types/hubot/index.d.ts

```diff
@@ -24,16 +25,17 @@ declare namespace Hubot {
         match: RegExpMatchArray;
         message: Message;
 
-        constructor(robot: Robot, message: Message, match: RegExpMatchArray);
+        constructor(robot: Robot<any>, message: Message, match: RegExpMatchArray);
```

Comment:
starting a review, so I get pinged when there's a change.

Reply:
@markspolakovs when you submit more changes I'll get pinged to review again... which I won't get to before Monday.